### ISSUE TITLE
Questions about the results in the paper

### ISSUE THREAD

**Issue by WideStars** [skingorz/FD-Align#1]
Hi, thanks for your great work! I have a few questions regarding your paper.

1. Do the results of the FD-Align used in your tables/figures (Table 1, 3, and Figure 5, 10) include SPC? (Since you only emphasized in Table 2 that FD-Align with SPC)
_I think the answer is yes._
2.  Which **dataset** was used for ablation, and which **visual backbone** was employed? (**in Table 4**)
  _I assume it was ImageNet with ViT-B/32, but I have some additional questions based on this assumption:_
    - The first line of the CLIP zero-shot result is 60.33, which is similar to the result obtained by using ResNet50 as the visual backbone with 7 prompts (in Tip-Adapter).
    - The second line of Fully Fine-tuning results seem to be the average results over 11 datasets shown in Figure 10, but ZS CLIP and FD-Align ablation results are not matchable if so. (By the way, the average results of FT are different in Figure 5 and Figure 10)
    - The results in the subsequent lines seem to be higher than expected for ViT-B/32, suggesting that ViT-B/16 might have been used instead?

![image](https://github.com/skingorz/FD-Align/assets/125349509/43897e4a-0a76-4169-9cdc-d82bb363d11b)

**Comment by skingorz** [comment]
Thanks for your attention to our work!
1. The results of FD-Align in our paper include SPC.
2. The results in Table 4 are evaluateed on the the average of 11 datasets and the results with ViT/32.

- In Figure 4, we used the same learning rate (5e-5) for all datasets in all methods for ablation experiments. FD-Align in Figures 5 and 10 report using the learning rate set by this repo, so FD-Aligns are not aligned in Table 4 and Figure 5, 10.

- The FT in Figure 10 uses the learning rate the same as the ablation experiment, but FD-Align uses the learning rate of this repo, while the FT in Figure 5 uses the learning rate of aligning the this repo with the FD Align. For a fair comparison, we will update the FT of Figure 10 to align with Figure 5.

3. All results in this paper are trained and evaluated on ViT/32.

4. Thank you for your reminder. There are indeed some errors in Figures 5 and 10.

- The average results of Figure 5 and Figure 10 should be 60.33 as described in Table 4, but we mistakenly plotted it as 63.03.

Thank you again for your detailed reading. We will update our paper on arxiv in the coming days.

Do you have any other questions？

**Comment by WideStars** [comment]
OK, thank you for your reply, it's much appreciated! 
I have another similar question regarding the results presented in Table 3.
_If I guess correctly, the dataset used is ImageNet, right?_

Additionally, I wanted to offer a friendly suggestion. 
When you update your paper in the future, it might be helpful to include a mention of the datasets used for any figures or tables that may not be immediately clear. This will provide readers with a better understanding of your paper. 
Thank you!

**Comment by skingorz** [comment]
Yes, the datasets in Table 3 is ImageNet. We will provide detailed explanations in subsequent revisions.
